### PR TITLE
Modified PR Template added

### PR DESCRIPTION
**Pull request by raashika03**
@bryan-brancotte  #73

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!--
+Thank you for your pull request!
+
+-->
+
+PR Checklist:
+
+- [x] Note the issue (if any) closed by this PR with [closing keywords](https://help.github.com/articles/closing-issues-using-keywords)
```
Comment (bryan-brancotte) [inline .github/PULL_REQUEST_TEMPLATE/pull_request_template.md:8]: checkboxs should not be checked

**Review by bryan-brancotte — CHANGES_REQUESTED** [review]

File: .github/PULL_REQUEST_TEMPLATE/pull_request_template.md (hunk 1 above)
Comment (raashika03) [reply .github/PULL_REQUEST_TEMPLATE/pull_request_template.md]: Done @bryan-brancotte

**Comment by raashika03** [comment]
should i add this line also. 
`Put an x in the boxes that apply`

**Comment by bryan-brancotte** [comment]
Was indeed needed ! I added content to the template, now merging

**Review by bryan-brancotte — APPROVED** [review]

**Comment by raashika03** [comment]
I was thinking to add few more points but was not sure if I should add. 
Now you added properly 🙏👏👏👏thank you @bryan-brancotte

**Pull request merged**